### ISSUE TITLE
Lightning Specification Meeting 2021/11/22

### ISSUE THREAD

**Issue by t-bast** [lightning/bolts#936]
The meeting will take place on Monday 2021/11/22 at 7pm UTC (5:30am Adelaide time) on Libera Chat IRC [#lightning-dev](irc://irc.libera.chat/lightning-dev). It is open to the public.

A video link is available if higher bandwidth communication is necessary at some point during the meeting: https://meet.google.com/abb-ekvg-oaw

## Pull Request Review

- [ ] ~Route blinding #765~
- [x] Onion messages #759
- [x] Add payment metadata to payment request #912
- [x] Advertize compression algorithms in `init` #825
- [x] Dynamic DNS support in gossip messages #911 or #917
- [ ] ~BLIPs #884~
- [x]  Simple turbo channels enablement #910
- [ ] ~Anchor outputs clean-up #903~
- [x] Feature bit for channel type #906
- [ ] ~Warning messages #834~
- [ ] ~Clarify `channel_reestablish` requirements #932~
- [ ] ~Add dust exposure threshold #919~
- [ ] ~Drop ping sending limitation #918~

## Long Term Updates

- [ ] ~Dual funding #851~
- [ ] ~Liquidity ads #878~
- [ ] ~Splicing #863~
- [ ] ~Offers #798~
- [ ] ~Trampoline routing #829 and #836~
- [ ] ~Upfront payments / DoS protection #843~
- [ ] ~Peer storage backup #881~

## Backlog

The following are topics that we should discuss at some point, so if we have time to discuss them great, otherwise they slip to the next meeting.

- [ ] ~lnprototest (https://github.com/rustyrussell/lnprototest)~

**Comment by michaelfolkson** [comment]
Hey @t-bast

> A video link is available if higher bandwidth communication is necessary at some point during the meeting:

If you want to try doing these meetings entirely on video (with someone monitoring IRC for additional comments/questions) I am happy to do a transcript like I've been doing for the [c-lightning dev meetings](https://btctranscripts.com/c-lightning/). Too late notice for today's meeting but something to consider for future meetings perhaps.

**Comment by t-bast** [comment]
Thanks @michaelfolkson, our main concern was indeed the lack of a transcript if we did the meeting over video, if you kindly propose to do it would be great! I'll mention it tonight and if we have rough agreement, we can try it for the next meeting.

**Comment by vincenzopalazzo** [comment]
>If you want to try doing these meetings entirely on video (with someone monitoring IRC for additional comments/questions) I am happy to do a transcript like I've been doing for the c-lightning dev meetings. Too late notice for today's meeting but something to consider for future meetings perhaps.

Concept ack

**Comment by lucasdcf** [comment]
> Hey @t-bast
> 
> > A video link is available if higher bandwidth communication is necessary at some point during the meeting:
> 
> If you want to try doing these meetings entirely on video (with someone monitoring IRC for additional comments/questions) I am happy to do a transcript like I've been doing for the [c-lightning dev meetings](https://btctranscripts.com/c-lightning/). Too late notice for today's meeting but something to consider for future meetings perhaps.

I was actually planning on proposing the same thing. I can help you if needed.

**Comment by t-bast** [comment]
One of my highlights from yesterday is that we're having issues agreeing on when/how feature bits and channel types should be used (in the context of #910, but it's something we bumped into a few times before).

I think it would be useful to focus on this particular topic during the next spec meeting, clarifying that meta topic will help speed up future discussions. We can use 0-conf as an example scenario to highlight our disagreements. @Roasbeef can you write something down to detail your approach (maybe in a new issue) and I'll do the same in parallel, and we can use that to guide the discussions next time?

**Comment by michaelfolkson** [comment]
Transcript (not yet merged but can be read here): https://github.com/michaelfolkson/bitcointranscripts/blob/lightning-2021-11-22/lightning-specification/2021-11-22-specification-call.md

**Comment by t-bast** [comment]
Thanks @michaelfolkson !
